### DISCUSSION TITLE
Cool readme contribution graph

### DISCUSSION THREAD

**Discussion by linonetwo** [TiddlyWiki/TiddlyWiki5#7979] [Ideas]
https://repobeats.axiom.co

see https://github.com/tiddly-gittly/TiddlyWiki-CPL?tab=readme-ov-file#tiddlywiki-cpl--- for an example

![Alt](https://repobeats.axiom.co/api/embed/c42c99e21691fa60ea61b5cdf11c2e0647621534.svg "Repobeats analytics image")

but only @Jermolene can create one

**Comment by Jermolene** [comment]
Interesting. I have connected things up to get this:

https://repobeats.axiom.co/api/embed/5a3bb51fd1ebe84a2da5548f78d2d74e456cebf3.svg

We'll need a PR to get into the readme
